package.json

```diff
@@ -52,6 +52,7 @@
     "react-router-dom": "^5.1.2",
     "react-transition-group": "^4.3.0",
     "reactfire": "^2.0.3",
+    "rxjs": "^6.5.5",
```

Comment:
Didn't see us use this anywhere, but if it was for debounce like you were mentioning before, we may want to use [lodash's debounce](https://lodash.com/docs/4.17.15#debounce) instead of pulling in rxjs just for that

Reply:
Correct, it's only for debouncing on the search.  I used this because reactfire is also using it.  I was assuming that this would only add the additional functions like debounce.  I didn't look into how much this increased the bundle size, though.